log_msg.log

```diff
@@ -0,0 +1,196 @@
+
```

Comment:
to be deleted. Log files must not be checked in.

Reply:
deleted